sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpMessageConverter.cs

```diff
@@ -219,6 +219,16 @@ public static ServiceBusReceivedMessage AmqpMessageToSBMessage(AmqpMessage amqpM
             {
```

Comment:
Do we trust that the message is well-formed and only has one body type?  Should we validate and fail if that invariant fails?